### PR TITLE
add pineapple casks

### PR DESCRIPTION
**Pull request by taylorterry3**
Adding casks for Pineapple, a Python 2/3 standalone version of ipython notebook

### CODE REVIEW

File: Casks/pineapple.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask :v1 => 'pineapple' do
+  version '0.7.4'
+  sha256 '6ef0bab79ff2a06da962dae5b8f68c3f6999140fd5f4253c4e1ef5524596cfc1'
+
+  # amazonaws.com is the official download host per the vendor homepage
+  url 'https://s3.amazonaws.com/com.nathansuniversity/Pineapple/Releases/Pineapple-Pro-0.7.4-Python2.7.dmg'
```
Comment (adityadalal924) [inline Casks/pineapple.rb]: should be versioned:
```
url "https://s3.amazonaws.com/com.nathansuniversity/Pineapple/Releases/Pineapple-Pro-#{version}-Python2.7.dmg"
```

**Comment by sammcj** [comment]
A couple of small 'offenses' failed the build there:

```
Offenses:

Casks/pineapple.rb:13:1: C: 1 trailing blank lines detected.
Casks/pineapple3.rb:13:1: C: 1 trailing blank lines detected.

2757 files inspected, 2 offenses detected

RuboCop failed!
```

File: Casks/pineapple.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  name 'Pineapple'
+  homepage 'http://nwhitehead.github.io/pineapple/'
+  license :gpl
+
+  app 'Pineapple-Pro.app'
+end
+
```
Comment (vitorgalvao) [inline Casks/pineapple.rb]: Extra end lines must be removed.

**Comment by vitorgalvao** [comment]
I’m a bit torn about including both of these here, but I’m going with “no”. Please make the one using python 3 the main one (since it’s the main one linked on the website) named `pineapple`, and make the other one `pineapple-python2` in [caskroom/versions](https://github.com/caskroom/homebrew-versions).

**Comment by taylorterry3** [comment]
Will clean up the above two errors. Re: having two versions, @vitorgalvao I'll do it that way if you want but the Python convention is usually `foo

**Comment by jawshooah** [comment]
I'm going to agree with @vitorgalvao here. While the convention in the Python community may be to assume compatibility with Python 2 by default, this particular app markets the version using Python 3 as the primary one.

**Comment by vitorgalvao** [comment]
Precisely what @jawshooah said. I was writing essentially the same thing but he beat me to the punch. The logic here is “follow the default”.

**Comment by taylorterry3** [comment]
Ok, cleaned all of that up. Thanks everyone for the help, sorry I suck at using my editor. Is the CI failure from having `#{version}` in the URL expected or is there a way around that?

**Comment by vitorgalvao** [comment]
You need to use double quotes (i.e. `url "https://s3.amazonaws.com/com.nathansuniversity/Pineapple/Releases/Pineapple-Pro-#{version}-Python3.5.dmg"`) or `#{version}` will be taken literally, instead of expanding (which is what we want).

**Comment by taylorterry3** [comment]
Duh. Thanks for the hand holding.

**Comment by vitorgalvao** [comment]
Thank you fro your patience and work on this. Forgot to mention something, though. The cask needs to be renamed to `pineapple-pro` with header `cask :v1 => 'pineapple-pro do`. The [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_token_reference.md) must be followed, to avoid duplicates.

**Comment by taylorterry3** [comment]
OK, we should be good.

**Comment by vitorgalvao** [comment]
We are, yes. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git) so we can merge?

**Comment by taylorterry3** [comment]
Done.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as dd8a4a61a309a6abc4e4a564ef1bfcbff6397384 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**